aws/resource_aws_ecs_service_test.go

```diff
@@ -1517,6 +1517,7 @@ resource "aws_ecs_service" "main" {
 	network_configuration {
 		security_groups = ["${aws_security_group.allow_all_a.id}", "${aws_security_group.allow_all_b.id}"]
 		subnets = ["${aws_subnet.main.*.id}"]
+		assign_public_ip = true
```

Comment:
Can you please add an appropriate attribute test where this is used (and one in the other Fargate test where it should be the default of false)? e.g. `resource.TestCheckResourceAttr("aws_ecs_service.main", "network_configuration.0.assign_public_ip", "true")`